readme.md

```diff
@@ -25,17 +25,17 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ## Checklist
 
-❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
+❗️ Replace `jacube1` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [jacube1](https://<your_account>.github.io/layout_moyo-header/)
+- [jacube1](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Need to fix this link and update it in the PR description
<img width="514" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/2ed50048-be94-40a0-b727-ddb5f782d3ef">
```suggestion
- [DEMO LINK](https://jacube1.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://jacube1.github.io/layout_moyo-header/report/html_report/)
```